docs/database-collection/time-series.txt

```diff
@@ -0,0 +1,186 @@
+add an ``up()`` function to specify the collection configuration.
+In the ``up()`` function, pass the new collection's name
+and the ``timeseries`` option to the ``Schema::create()`` method.
+
+.. tip::
+
+   To learn more about creating a migration class, see :ref:`laravel-eloquent-migrations`
+   in the Schema Builder guide.
+
+When setting the ``timeseries`` option, include the following fields:
+
+- ``timeField``: Specifies the field that stores a timestamp in each time series document.
+- ``metaField``: Specifies the field that stores metadata in each time series document.
+- ``granularity``: Specifies the approximate time between consecutive timestamps. The possible
+  values are ``'seconds'``, ``'minutes'``, and ``'hours'``.
+
+.. _laravel-time-series-create-example:
+
+Example
+~~~~~~~
+
+This example migration class creates the ``precipitation`` time series collection
+with the following configuration:
+
+- ``timeField`` is set to ``'timestamp'``
+- ``metaField`` is set to ``'location'``
+- ``granularity`` is set to ``'minutes'``
+
+.. literalinclude:: /includes/database-collection/time-series-migration.php
+   :language: php
+   :dedent:
+
+To verify that you successfully created the time series collection, call
+the ``Schema::hasCollection()`` method and pass the collection name as
+a parameter:
+
+.. code-block:: php
+
+   $result = Schema::hasCollection('precipitation');
+   echo $result;
+
+The ``hasCollection()`` method returns a value of ``1`` if the
+collection exists.
+
+.. _laravel-time-series-insert:
+
+Insert Time Series Data
+-----------------------
+
+You can insert data into a time series collection by passing your documents to the ``insert()``
+method and specifying the measurement, timestamp, and metadata in each inserted document.
+
+.. tip::
+
+   To learn more about inserting documents into a collection, see :ref:`laravel-fundamentals-insert-documents`
+   in the Write Operations guide.
+
+Example
+~~~~~~~
+
+This example inserts New York City precipitation data into the ``precipitation``
+time series collection created in the :ref:`Create a Time Series Collection example
+<laravel-time-series-create-example>`. Each document contains the following fields:
+
+- ``precipitation_mm``, which stores precipitation measurements in millimeters
+- ``location``, which stores location metadata
+- ``timestamp``, which stores the time of the measurement collection
+
+.. code-block:: php
+
+    $data = [
+        [
+            'precipitation_mm' => 0.5,
+            'location' => 'New York City',
+            'timestamp' => new UTCDateTime(Carbon::create(2023, 9, 12, 0, 0, 0, 'CET')),
+        ],
+        [
+            'precipitation_mm' => 2.8,
+            'location' => 'New York City',
+            'timestamp' => new UTCDateTime(Carbon::create(2023, 9, 17, 0, 0, 0, 'CET')),
+        ],
+    ];
+
+    $result = DB::table('precipitation')
+        ->insert($data);
```

Comment:
It would be useful to have this example extracted with a test, to ensure there is no error.